### ISSUE TITLE
Linter stuck, only opening and closing file resolves error

### ISSUE THREAD

**Issue by samblyon** [AtomLinter/linter-eslint#1151]
## Issue Type
Bug

## Issue Description
As I code, I frequently get `Parsing Error: Unexpected token, expected "}" (fatal)`. This error makes sense if I haven't closed a closure, etc. The issue is even after I complete my closure, the error persists. Running the `eslint` cli shows no errors, and closing and reopening the file clears the error. This is a recent problem.. my guess is eslint is overwhelmed with linting files, however I'm not sure how to debug that. There doesn't seem to be a way to "re-lint" as a way to check.

E.g.
```
import {
``` 
Wait a second, lint error shows up.

```
import map from 'lodash/map';
``` 
Error still says unexpected token. Running eslint cli shows no error. Close file, open file, error disappears.

I've noticed even after I close a file, the atom lint status still shows that file under "Current"

## Bug Checklist
- [X] Restart Atom
- [X] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [X] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.28.2
linter-eslint version: 8.4.1
ESLint version: 3.3.0
Hours since last Atom restart: 9.7
Platform: darwin
Using local project ESLint from: /srv/nerdwallet/react-local-dev/packages/my-nerdwallet/node_modules/eslint
Current file's scopes: [
  "source.js.jsx",
  "meta.class.js",
  "entity.other.inherited-class.js"
]
linter-eslint configuration: {
  "fixOnSave": true,
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDirs": [],
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": [],
  "ignoreFixableRulesWhileTyping": false
}
```

.eslintrc

```
{
  "extends": ["@company/eslint-config", "airbnb", "plugin:jsx-control-statements/recommended"],
  "extends": ["@company/eslint-config", "airbnb"],
  "plugins": ["jsx-control-statements"],
  "rules": {
    "react/jsx-no-bind": "off",
    "no-return-assign": "off",
    "react/sort-comp": "off",
    "react/no-multi-comp": "off",
    "react/no-find-dom-node": "off",
    "jsx-a11y/img-has-alt": "off",
    "jsx-a11y/label-has-for": "off",
    "jsx-a11y/heading-has-content": "off",
    "jsx-control-statements/jsx-jcs-no-undef": "error",
    "jsx-control-statements/jsx-use-if-tag": "off",
    "no-script-url": "off",
    "import/no-named-as-default": "off",
    "import/no-extraneous-dependencies": "off",
    "prettier/prettier": "off"
  },
  "settings": {
    "import/resolver": {
      "webpack": {
        "config": "webpack.browser.config.js"
      }
    }
  }
}
```

**Comment by skylize** [comment]
Are you still having this issue? Seems like it may be sort-of a duplicate of #927.

**Comment by dcporter44** [comment]
I'm having this issue as well. It seems sudden. It's been working fine for me for probably 2 years. Maybe I installed something that is interfering... I'm not sure.

When I get an eslint error and then fix it, the linter error remains. The code proceeds to compile just fine. It's just the linter error in the text-editor. If I close that file and re-open it, it goes away. It's very annoying and I'm not sure what to do. I'm on the verge of switching from Atom.
